recipe/meta.yaml

```diff
@@ -104,8 +96,8 @@ outputs:
         - openssl
```

Comment:
All the following can be removed.
These are dependencies from libmamba, not listed in https://github.com/mamba-org/mamba/blob/2023.02.03/libmambapy/CMakeLists.txt
        - openssl
        - yaml-cpp
        - cpp-expected
        - spdlog 1.11.0
        - fmt 9.1.0
        - termcolor-cpp
        - nlohmann_json

Reply:
Removing any of these except for `termcolor-cpp` caused build failures. I suspect `libmambapy` pulls its dependency list from `libmamba`, but I could be wrong.